artiq/coredevice/shuttler.py

```diff
@@ -2,20 +2,54 @@
 
 from artiq.language.core import *
 from artiq.language.types import *
-from artiq.coredevice.rtio import rtio_output
+from artiq.coredevice.rtio import rtio_output, rtio_input_data
+from artiq.coredevice import spi2 as spi
+from artiq.language.units import us
+
+
+@portable
+def shuttler_volt_to_mu(volt):
+    return round((1 << 14) * (volt / 20.0)) & 0x3fff
```

Comment:
This should be directly usable when programming the DC bias `Volt` waveform, but `CORDIC_GAIN` needs to be factored in for the `DDS` waveform component.

I might just add a comment about it and leave it there.